distribution/docs/src/main/resources/content/_sources/solr-catalog-provider.adoc

```diff
@@ -11,153 +11,88 @@
+* Backup can be performed using ${branding} console's `backup` command.
+* This configuration cannot be scaled larger than the single Solr server.
+* All data is located inside the {$branding} home directory. If the Solr index grows
+  large, the storage volume may run low on space.
 
-<<_solr_cloud,Solr Cloud>>:: Used with a cluster of Solr servers.
+.Installing Internally Managed Solr
+No installation is required because ${branding} includes a distribution of Apache Solr.
 
-If not using Solr Cloud, whenever the {branding} Solr ingests data, it creates an index for it in the data directory.
-This data indexing enables the data to be searchable by the {branding} Solr. The data in the data directory can then be persisted by performing a `commit` on the {branding} Solr.
-When the {branding} Solr is restarted, it reloads the data to get back the index.
+.Configuring Internally Managed Solr
+Although the default settings should be sufficient for most purposes, there are three options that
+can be configured for an internally managed Solr.
 
-.[[_http_solr]]HTTP Solr
-The HTTP Solr is an external Apache Solr server instance.
-There are two possible installations for the HTTP Solr configuration:
-
-Internal (default):: Points to an Apache Solr server that runs in the same container as {branding}.
-External:: Points to an Apache Solr server in a different container or on a different system.
-
-.HTTP Solr Internal Deployment
-[ditaa,http-solr-internal-diagram,png]
-....
- Container (JVM)
- +=----------------+
- |                 |
- | /-----\         |
- | |cDEF |         |
- | | DDF +--\      |
- | |     |  |http  |
- | \-----/  |      |
- |          |      |
- |          v      |
- |      +--------+ |
- |      |SolrcFF0| |
- |      +--------+ |
- |                 |
- +-----------------+
-....
-
-.HTTP Solr External Deployment
-[ditaa,http-solr-external-diagram,png]
-....
- Container (JVM)
- +=----------------+
- |                 |
- | /-----\         |
- | |cDEF |         |  +--------+
- | | DDF +----------->|SolrcFF0|
- | |     |   http  |  +--------+
- | \-----/         |
- |                 |
- +-----------------+
-....
-
-.When to Use HTTP Solr
-The HTTP Solr configuration is good for most use cases. See below for details.
-
-.HTTP Solr Tradeoffs
-* Solr Admin UI can be used.
-* Backup can be performed manually or using the Backup command.
-* External installation allows isolation of data storage.
-* Requires additional protection to be secured.
-* Not scalable.
-* External installation has network overhead
-
-.Installing HTTP Solr
-
-By default, the ${ddf-solr} application installs a Solr server inside the same container as {branding}.
-To install an External Solr server, refer to https://cwiki.apache.org/confluence/display/solr/Apache+Solr+Reference+Guide[Apache Solr Reference Guide].
-
-.Configuring HTTP Solr
+* Enable TLS. By default, Solr does not use transport layer security (TLS) because ${branding}
```

Comment:
Capitalize to "Transport Layer Security" to go with acronym.

Reply:
Enable TLS may need to be a title here instead of the start of a bullet.